static/app/bootstrap/initializeSdk.tsx

```diff
@@ -78,6 +78,27 @@ export function initializeSdk(config: Config, {routes}: {routes?: Function} = {}
       : sentryConfig?.whitelistUrls,
     integrations: getSentryIntegrations(hasReplays, routes),
     tracesSampleRate,
+    beforeSend(event, hint) {
+      /**
+       * There is a bug in iOS, that causes `AbortError` when fetch is aborted, and you are in the middle of reading the response.
+       * In Chrome and other browsers, it is handled gracefully, where in Safari on iOS, it produces additional error, that is jumping
+       * outside of the original Promise chain and bubbles up to the `unhandledRejection` handler, that we then captures as error.
+       * Can be safely removed once the bug is fixed upstream.
+       *
+       * Ref: https://bugs.webkit.org/show_bug.cgi?id=215771
```

Comment:
Wouldn't we continue to need this patch as there will be old Safari versions in the wild for quite some time.

Reply:
Kinda yes, but because people usually updates their ios quite often, it should be fine to remove it after a while :3